tgui/packages/tgui/interfaces/MafiaPanel.tsx

```diff
@@ -149,7 +188,11 @@ const MafiaLobby = (props, context) => {
         valid players signed up.
       </NoticeBox>
       {lobbydata?.map((lobbyist) => (
```

Comment:
won't block on this, but some prefer destructuring in the jsx block. it would look like this 
`lobbydata.map({ name, status, spectating }) =>`
(you also won't need to use an optional chain with my previous suggestion)